packages/flutter_test/lib/src/matchers.dart

```diff
@@ -1090,3 +1102,75 @@ class _ClipsWithShapeBorder extends _MatchRenderObject<RenderClipPath, Null> {
   Description describe(Description description) =>
     description.add('clips with shape: $shape');
 }
+
+class _CoversSameAreaAs extends Matcher {
+  _CoversSameAreaAs(
+    this.expectedPath, {
+      @required this.areaToCompare,
```

Comment:
trivial nit: we normally don't indent any more than the positional args here (for consistency with when there's no positional args)

Reply:
done